src/Dialogs/Device/DeviceListDialog.cpp

```diff
@@ -385,6 +389,12 @@ DeviceListWidget::OnPaintItem(Canvas &canvas, const PixelRect rc, unsigned idx)
     if (flags.traffic)
       buffer.append(_T("; FLARM"));
 
+    if (flags.temperature)
+      buffer.append(_T("; Temperature"));
+
+    if (flags.humidity)
+      buffer.append(_T("; Humidity"));
```

Comment:
I'm not too sure about this. If we would add every datapoint we receive here this will result in a looong string... 🤔

Reply:
I actually had that same thought back when I implemented it. I tried `; T` and `; H`, which looked a bit cryptic.